src/common/sound.cpp

```diff
@@ -33,16 +35,16 @@ const eSOUND_TYPE Sound::assistTypes[] = { eSOUND_TYPE::Start, eSOUND_TYPE::Butt
     eSOUND_TYPE::WaitingBeep };
 
 /// signals repeats - how many times will sound signals repeat (-1 is infinite)
-const int Sound::onceRepeats[] = { 1, 1, 1, 1, 1 };
```

Comment:
Please check my previous value from my next PR. There is good amount of changes discussed with testing department. viz. #994